harper-core/dictionary.dict

```diff
@@ -20894,6 +20897,8 @@ downpour/14MS
 downrange/5
 downright/5
 downriver/5
+downsample/
```

Comment:
I could use some direction on how to specify the affixes here. 

I found the [affixes.json](https://github.com/automattic/harper/blob/301215ef91b3caaf29a6/harper-core/affixes.json) file but I am struggling a bit at defining them myself. 

Is there something I should read before jumping in like this?